### ISSUE TITLE
Allow for multiple state definitions in one line (states :sleeping, :running, :cleaning)

### ISSUE THREAD

**Issue by rousisk** [aasm/aasm#146]
This is a feature request for some syntactic sugar. 

I would like to have an elegant way of adding multiple "vanilla" states in one line. And by "vanilla" I mean any state which isn't initial or has callbacks associated with it.

Example usage of the proposed feature: 

```ruby
aasm do
  state :open, initial: true
  states :processed, :cancelled, :flagged, :succeeded, :failed
  # ...
end
```

If you like the idea and you think it might be of value to more people I would like to prepare a PR.

**Comment by alto** [comment]
@rousisk Take a look at #288, @HParker gave it a try.

**Comment by alto** [comment]
I just released version `4.10.0` including the changes from #288.
